src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -130,13 +130,16 @@
            destination])
 
     "mention"
-    (conj acc [react/text-class
-               {:style    {:color (cond
-                                    (= content-type constants/content-type-system-text) colors/black
-                                    :else                                               colors/mention-incoming)}
-                :on-press (when-not (= content-type constants/content-type-system-text)
-                            #(re-frame/dispatch [:chat.ui/show-profile literal]))}
-               [mention-element literal]])
+    (conj acc
+          [react/view {:style {:background-color "rgba(67,96,223,0.1)" :border-radius 6 :padding-horizontal 3}}
+           [react/text-class
```

Comment:
same for color and `font-weight` should be used from quo2 foundation

Reply:
Didn't find any font-weight in quo2 foundation